theatre/studio/src/propEditors/useEditingToolsForSimpleProp.tsx

```diff
@@ -158,101 +164,85 @@ function createDerivation<T extends SerializablePrimitive>(
         })
 
         const sequenceTrackId = possibleSequenceTrackId as SequenceTrackId
-        const nearbyKeyframes = prism.sub(
-          'lcr',
-          (): NearbyKeyframes => {
-            const track = val(
-              obj.template.project.pointers.historic.sheetsById[
-                obj.address.sheetId
-              ].sequence.tracksByObject[obj.address.objectKey].trackData[
-                sequenceTrackId
-              ],
-            )
-            const sequencePosition = val(
-              obj.sheet.getSequence().positionDerivation,
+        const nearbyKeyframesD = prism(() =>
+          prism
+            .memo(
+              'lcr',
+              (): IDerivation<NearbyKeyframes> =>
+                prism(() => {
```

Comment:
prism of prism.memo of prism?? Can't wrap my brain around why it is like this. Can things be pulled out or something?